Task1/transportation.js

```diff
@@ -0,0 +1,17 @@
+//4 -http://www.codewars.com/kata/transportation-on-vacation
+function rentalCarCost(d) {
+    var noOfDays=d;
+    var amount;
```

Comment:
This is the problem statement for this method:
Every day you rent the car costs $40. If you rent the car for 7 or more days, you get $50 off your total. Alternatively, if you rent the car for 3 or more days, you get $20 off your total.

you haven't checked for >=7 days

you took 3 separate if statements
one for <=2, >=3 && <=6 , >=5
can you re-implement this method based on the problem statement

Let me know if you are facing any difficulties

first check for >=7 then check of >=3 (use if else ladder).